### PR TITLE
MNEMONIC-519: Add test cases for mapred based older Hadoop API

### PR DESCRIPTION
**Pull request by djkevincr**
Added chunk data, long data and person data test cases for mapred based following APIs,
1. MneInputFormat
2. MneMapredRecordReader
3. MneMapredRecordWriter
4. MneOutputFormat
These test cases are based on the test cases for mapreduce new APIs. However rewritten due to API changes.

### CODE REVIEW

File: mnemonic-hadoop/mnemonic-hadoop-mapreduce/src/test/java/org/apache/mnemonic/mapred/MneMapredBufferDataTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@
   @BeforeClass
   public void setUp() throws IOException {
     m_workdir = new Path(
-        System.getProperty("test.tmp.dir", DEFAULT_WORK_DIR));
+            System.getProperty("test.tmp.dir", DEFAULT_WORK_DIR));
```
Comment (bigdata-memory) [inline mnemonic-hadoop/mnemonic-hadoop-mapreduce/src/test/java/org/apache/mnemonic/mapred/MneMapredBufferDataTest.java]: looks the IDE adds those extra white-spaces, could you please remove that ? and there are more lines below, thanks.

Comment (djkevincr) [reply mnemonic-hadoop/mnemonic-hadoop-mapreduce/src/test/java/org/apache/mnemonic/mapred/MneMapredBufferDataTest.java]: +1, I will fix those.

Comment (djkevincr) [reply mnemonic-hadoop/mnemonic-hadoop-mapreduce/src/test/java/org/apache/mnemonic/mapred/MneMapredBufferDataTest.java]: @bigdata-memory Addressed the requested changes. Please check.

**Comment by bigdata-memory** [comment]
Sorry I have merged the MNEMONIC-520 first, so could you please adjust this commit based on the latest commit of upstream? thanks.

**Comment by djkevincr** [comment]
@bigdata-memory Sure I will do it now.

**Comment by djkevincr** [comment]
@bigdata-memory Please check now?

**Pull request merged**